app/routes/circulars.correction.$circularId.tsx

```diff
@@ -41,7 +41,7 @@ export async function loader({
     circularId: circular.circularId,
     defaultSubmitter: circular.submitter,
     defaultCreatedOnDate: defaultDateTime[0],
-    defaultCreatedOnTime: defaultDateTime[1].substring(0, 5),
+    defaultCreatedOnTime: defaultDateTime[1].replace(/\.\d{3}Z$/, ''),
```

Comment:
Hmm, so we're dropping millisecond precision here?

Reply:
I don't see a way to preserve millisecond precision while using `<input type="time">`. Perhaps we should just input the date and time as an ISO 8601 string?